docker-compose.pinner.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  pinner:
+    # uncomment "build" and comment out "image" to build from sources
+    # build: https://github.com/SkynetLabs/pinner.git#main
+    image: skynetlabs/pinner:0.1.0
+    container_name: pinner
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - PINNER_LOG_LEVEL=${PINNER_LOG_LEVEL:-trace}
```

Comment:
is there any specific reason why we're overriding a default log level with trace here?

Reply:
Yes. I want to default to a lot of output for this service, so we can monitor the way it works. It's both important, untested, and capable of causing a lot of damage, should there be a nasty bug. We can always override this with an env var.

Reply:
I would think we would want to do it other way around - set the env var initially to trace instead of defaulting to it for everyone but if that's temporary then I think it's ok. Maybe we can leave a note in the code about it so we remember to tune it down in some time?